src/handlers/sdp/MediaSection.ts

```diff
@@ -691,6 +691,52 @@ export class OfferMediaSection extends MediaSection
 			? encoding.rtx.ssrc
 			: undefined;
 
+		for (const codec of offerRtpParameters!.codecs)
+		{
+			const rtp: any =
+				{
+					payload : codec.payloadType,
+					codec   : getCodecName(codec),
+					rate    : codec.clockRate
+				};
```

Comment:
```suggestion
			const rtp: any =
			{
				payload : codec.payloadType,
				codec   : getCodecName(codec),
				rate    : codec.clockRate
			};
```

Pretty sure we use this syntax everywhere :)